config/dependency/envoy-gateway/kustomization.yaml

```diff
@@ -1,7 +1,12 @@
 kind: Kustomization
 
 resources:
-- https://github.com/envoyproxy/gateway/releases/download/v1.1.0/install.yaml
+- https://github.com/envoyproxy/gateway/releases/download/v1.2.8/install.yaml
 
 patches:
-- path: envoy_config_patch.yaml
\ No newline at end of file
+- path: envoy_gateway_patch.yaml
+
+images:
+- name: envoyproxy/gateway
+  newName: envoyproxy/gateway
+  newTag: v1.2.8
```

Comment:
please copy latest images to VKE container registry store

Reply:
this is done. I tested with new images.